### ISSUE TITLE
Working with API - no results

### ISSUE THREAD

**Issue by vpratfr** [alexurquhart/vue-bootstrap-typeahead#15]
There should be a way to give the user some feedback when there is no match from the API.

Currently we only see the shadow of the result list.

**Comment by alexurquhart** [comment]
Good idea, I think either having a prop or custom slot (or both) to allow placeholder info would be useful. Will also allow users to give some additional feedback on loading state if there's an API call happening

**Comment by kyleweishaupt** [comment]
A slot for empty results would be super helpful to me. Really hoping this package gets updated sometime soon. It's the best solution I've found for an autocomplete with Bootstrap and Vue. 

If anyone has a workaround for now I would greatly appreciate it.

**Comment by inarucast** [comment]
Any update with this improvement?

**Comment by wpgeek922** [comment]
Hi, I have the same requirement. There must be some way to convey users for no matches or empty results. Is there any update on the above issue?

**Comment by wpgeek922** [comment]
I have pushed fix for the above issue. Accept my PR and merge it in to master.
